test/napi/lib/strings.js

```diff
@@ -5,6 +5,14 @@ describe("JsString", function () {
   it("should return a JsString built in Rust", function () {
     assert.equal(addon.return_js_string(), "hello node");
   });
+  describe("encoding", function () {
+    it("should return the UTF-8 string length", function () {
+      assert.equal(addon.return_length_utf8("a🥹"), 5);
+    });
+    it("should return the UTF-16 string length", function () {
```

Comment:
I appreciate all these great tests and this isn't necessary, but I think it would be valuable to have a test as an example that uses `TextDecoder` (or `Buffer`) to decode UTF-16 to a string.